homeassistant/components/duckdns/__init__.py

```diff
@@ -85,7 +94,42 @@
     body = await resp.text()
 
     if body != "OK":
+        _LOGGER.debug("time %s", dt_util.utcnow())
```

Comment:
all logging is timestamped, this is not needed.

Reply:
Will delete this...